cdap-cli/src/main/java/io/cdap/cdap/cli/command/ConnectLinkCommand.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright © 2020 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.cdap.cli.command;
+
+import io.cdap.cdap.cli.ArgumentName;
+import io.cdap.cdap.cli.CLIConfig;
+import io.cdap.cdap.cli.CLIConnectionConfig;
+import io.cdap.cdap.cli.LaunchOptions;
+import io.cdap.cdap.cli.util.InstanceURIParser;
+import io.cdap.common.cli.Arguments;
+import io.cdap.common.cli.Command;
+
+import java.io.PrintStream;
+import javax.inject.Inject;
+
+/**
+ * Connects to a CDAP instance.
```

Comment:
How does it differ from ConnectCommand? May be a bit more information like "Connects to a CDAP instance using given URI. URI can be in format of ..."